server/planning/feed_parsers/superdesk_event_json.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import datetime
+
+from superdesk.io.feed_parsers import FileFeedParser
+import pytz
+import json
+
+utc = pytz.UTC
+logger = logging.getLogger(__name__)
+
+
+class SuperdeskFeedParser(FileFeedParser):
+    """Superdesk event specific parser.
+
+    Feed Parser which can parse the Superdesk Event feed and convert to internal event format,
+    but the firstcreated and versioncreated times are localised.
+    """
+
+    NAME = 'SuperdeskEventJson'
```

Comment:
This name is more of a code than a name (i.e. ICS2.0 uses `ics20`).
Something like `json_event` would be in line with the other parsers